org-super-agenda.el

```diff
@@ -302,15 +305,27 @@ marker."
   "Global minor mode to group items in Org agenda views according to `org-super-agenda-groups'.
 With prefix argument ARG, turn on if positive, otherwise off."
   :global t
-  (let ((advice-function (if org-super-agenda-mode
-                             (lambda (to fn)
-                               ;; Enable mode
-                               (advice-add to :filter-return fn))
-                           (lambda (from fn)
-                             ;; Disable mode
-                             (advice-remove from fn)))))
-    (funcall advice-function #'org-agenda-finalize-entries
+  (let ((advice-function-filter-return (if org-super-agenda-mode
+                                           (lambda (to fn)
+                                             ;; Enable mode
+                                             (advice-add to :filter-return fn))
+                                         (lambda (from fn)
+                                           ;; Disable mode
+                                           (advice-remove from fn))))
+        (advice-function-after (if org-super-agenda-mode
+                                   (lambda (to fn)
+                                     ;; Enable mode
+                                     (advice-add to :after fn))
+                                 (lambda (from fn)
+                                   ;; Disable mode
+                                   (advice-remove from fn)))))
+    (funcall advice-function-filter-return #'org-agenda-finalize-entries
              #'org-super-agenda--filter-finalize-entries)
+    (funcall advice-function-after #'org-agenda-filter-apply
+             #'org-super-agenda--hide-or-show-groups)
+    (funcall advice-function-after #'org-agenda-finalize
+             #'org-super-agenda--hide-or-show-groups)
+
```

Comment:
I would prefer to reduce the number of advices to the minimum, if possible.  I haven't studied the code in question, so it may not be possible to reduce this further.  What do you think?

Reply:
As far as I know, it's not possible to further reduce it. But I could/will ask that question on the org-devel mailing list.

Reply:
I didn't get any response!